formats/filtered/src/Filtered.php

```diff
@@ -164,6 +165,7 @@ protected function getResultText( SMWQueryResult $res, $outputmode ) {
 		$result = [];
 		while ( $row = $res->getNext() ) {
 			$result[ $this->uniqid() ] = new ResultItem( $row, $this );
+			usleep( 1 );
```

Comment:
Yes, it is always the same query. `mResults` contains 50 entries, but `next` returns `false` too early. Tried resetting `mResults` (which as expected did not help, as it  is the first few entries that are returned. It does not *start* somewhere in between, but *stop* too early), tried replacing `current` + `next` by `each`, even tried not using `getNext` at all, but copy/pasting its content into my loop directly. No dice.